infra/cifuzz/example_main.yml

```diff
@@ -1,12 +1,8 @@
 name: CIFuzz
-
-on: [push]
-
+on: [pull_request]
```

Comment:
Why are we switching this?

Reply:
Push is used for testing. 'pull_request' is used in production. In my documentation I told people to copy this file so it would be good to have the production version.

Reply:
ACK